### PR TITLE
Integration with Quill Editor

### PR DESCRIPTION
**Pull request by camilleriluke**
Related to issue: https://github.com/codebuddiesdotorg/cb-v2-scratch/issues/196

### Changes
- Introduced new module `QuillEditor` (internally it works with [QuillJS](http://quilljs.com/)). 
    + For more information on how it works, look at the tests
- All `textarea` instances to show the hangout description are migrate to use this mechanism
- This module also comes with its own test suite
    + To run tests: `meteor test --driver-package practicalmeteor:mocha --port 3100`
- Editor is backward compatible with old data
- Server validation to accept the new hangout description format

### Notes
- Hangout description will now be saved using Deltas. Read more: [https://quilljs.com/guides/working-with-deltas/](https://quilljs.com/guides/working-with-deltas/).
Old data will be converted to this format on load time and saved in the new format on next save (which makes this feature backward compatible)
- Run `npm install` before running this branch  since a couple of new dependencies where added.

/cc @lpatmo @distalx

### CODE REVIEW

File: server/hangouts/methods.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ Meteor.methods({
     check(data, Match.ObjectIncluding({
       topic: String,
       slug: String,
-      description: String,
+      description: Match.ObjectIncluding({
```
Comment (camilleriluke) [inline server/hangouts/methods.js]: I'm not totally sure about this `Math.Any`.

The reason for this change is that description will now be submitted in this format https://quilljs.com/guides/working-with-deltas/

**Comment by lpatmo** [comment]
Wow, awesome -- thanks for your hard work, @camilleriluke! Will take a careful look through the code.

**Comment by lpatmo** [comment]
This is perfect, @camilleriluke! :) (Sorry it took me a while to review.)

**Comment by lpatmo** [comment]
One thing -- not really related to this issue itself -- but noticed ```import QuillEditor from '../../libs/QuillEditor';

Template.hangoutActionBar.helpers({
  create: function(){

  },
  rendered: function(){
    
  },
  destroyed: function(){

  },
});``` in hangout_action_bar.js, and I think we can remove it.

**Comment by camilleriluke** [comment]
Thanks @lpatmo and @distalx for helping me out and giving me suggestions. 

@lpatmo remember that this PR is based on `issue-196` (your branch) so it would need another PR to hit master.

**Pull request merged**